### PR TITLE
Correct intro.rst with regards to required Python version

### PR DESCRIPTION
**Pull request by jdufresne**
(Sorry, I missed this one in #5477.)

### CODE REVIEW

File: doc/intro.rst [hunk 1]
```diff
@@ -55,10 +55,10 @@ See the :ref:`pertinent section in the FAQ list <usingwith>`.
 Prerequisites
 -------------
 
-Sphinx needs at least **Python 2.7** or **Python 3.4** to run, as well as the
-docutils_ and Jinja2_ libraries.  Sphinx should work with docutils version 0.10
-or some (not broken) SVN trunk snapshot.  If you like to have source code
-highlighting support, you must also install the Pygments_ library.
+Sphinx needs at least **Python 3.5** to run, as well as the docutils_ and
+Jinja2_ libraries. Sphinx should work with docutils version 0.10 or some (not
```
Comment (tk0miya) [inline doc/intro.rst]: Since 2.0, we updated requirement to docutils-0.12 or above.

File: doc/intro.rst [hunk 2]
```diff
@@ -55,10 +55,10 @@ See the :ref:`pertinent section in the FAQ list <usingwith>`.
...
+broken) SVN trunk snapshot. If you like to have source code highlighting
+support, you must also install the Pygments_ library.
```
Comment (tk0miya) [inline doc/intro.rst]: Now pygments has been added to requirements list. So this sentence is not needed!

**Review by tk0miya — CHANGES_REQUESTED** [review]
Could you also updated related text?

**Comment by jdufresne** [comment]
Done. Let me know if that is what you had in mind. I can do any other edits if needed.

**Review by tk0miya — APPROVED** [review]

**Comment by tk0miya** [comment]
Thank you always :-)

**Pull request merged**